<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Codificador</class>
 <widget class="QWidget" name="Codificador">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Codificador</string>
  </property>
  <widget class="QPushButton" name="btnCode">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Codificar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDecode">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Decodificar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGetSourceFile">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txtInput">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Carregar arquivo .csv com colunas a serem codificadas</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblResult">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txtSeparator">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="plainText">
    <string>;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSeparator">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Separador</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbModo">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Base64 (simples)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Vigenere</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fenet</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lblModo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Modo</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblFonte">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Fonte</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblDestino">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Destino</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txtOutput">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSetOutputFile">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>80</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
